server/routes/profile.js

```diff
@@ -1,9 +1,28 @@
 const express = require('express');
-
+const mongoose = require('mongoose');
 const router = express.Router();
 
+const Tracker = require('../models/tracker');
+
 router.get('/check', async (req, res) => {
   res.json({ success: true })
 });
 
+router.get('/chart/:id', async (req, res) => {
+  var time = Number(req.query.date);  
+  let userId = mongoose.Types.ObjectId(req.params.id); 
```

Comment:
userId from req.user._id

Reply:
/profile/chart/<<userId>> : this is the input.

https://github.com/vnts7/mwa-prj/pull/13/commits/a26e031c064bb230bba9507b1e3044225edc2304  Chart2Component get userId to retrieve data